client/client_pact_test.go

```diff
@@ -53,8 +53,8 @@ func TestTerraformClientPact(t *testing.T) {
 					b.Header("Authorization", Like("Bearer 1234"))
 					b.JSONBody(Like(pacticipant))
 				}).
-				WillRespondWith(200, func(b *consumer.V2ResponseBuilder) {
-					b.Header("Content-Type", S("application/hal+json"))
+				WillRespondWith(201, func(b *consumer.V2ResponseBuilder) {
+					b.Header("Content-Type", S("application/hal+json;charset=utf-8"))
```

Comment:
Is the lack of `charset` causing issues in verification? It's not strictly needed by the client.

This being said, the header claim is not ideal - it's not actually checking for a JSON type, it's just expecting a string. It could be switched to a regex to ensure it starts with `application/json`.